projects/elements-demo/src/app/app.module.ts

```diff
@@ -25,7 +26,11 @@ import { environment } from '../environments/environment';
     ServiceWorkerModule.register('ngsw-worker.js', {
       enabled: environment.production
     }),
-    LazyElementsModule,
+    LazyElementsModule.forRoot({
+      rootOptions: {
+        errorComponent: GlobalErrorComponent
```

Comment:
`RootErrorComponent`, please we should keep it consistent

Reply:
There should be no `global` used anywhere as we spoke about, consistency makes a big difference for this kind of libs